app/pages/schedule.vue

```diff
@@ -22,3 +122,180 @@ useSeoMeta({
     ogImage: computed(() => prismic.asImageSrc(page.value?.data.meta_image)),
 });
 </script>
+
+<style lang="postcss" scoped>
+.page-container {
+    max-width: 1240px;
```

Comment:
Peut être mettre dans une variable css `--page-container-max-width: 1240px;` si la même valeur est appliquée dans d'autre page
```suggestion
    max-width: var(--page-container-max-width);
```